### DISCUSSION TITLE
navbar:item link not dynamic after versioning

### DISCUSSION THREAD

**Discussion by Sotty75** [facebook/docusaurus#5577] [Q&A] [answered]
Hi,
i have versioned my docusaurus site with two versions:

- next
- 3.10.2

but the navbar item pointing to may docs folder is still static.
The link is still:
http://localhost:3000/docs/

while i would expect it to change to 

http://localhost:3000/docs/next/
http://localhost:3000/docs/version-3.10.2/

the navbar is configred as follows:

```
navbar: {
            title: 'Devex for Developers',
            logo: {
                alt: 'Devex Developers Documentation',
                src: 'img/Selerant-logo-swirl.svg',
            },
            items: [
                {
                    to: 'docs',
                    activeBasePath: 'System Requirements',
                    label: 'Administrative Guide',
                    position: 'left',
                },
                {
                    to: 'release-doc',
                    label: 'Release Documentation',
                    position: 'left',
                },
                {
                    type: 'docsVersionDropdown',
                    position: 'right',
                },
                { to: 'blog', label: 'Team Blog', position: 'right' },
            ],
        },
```

while the docs section as follows:

```
presets: [

        [
            '@docusaurus/preset-classic',
            {
                docs: {
                    sidebarPath: require.resolve('./sidebars.js'),
                    // Please change this to your repo.
                    editUrl:
                        'https://github.com/facebook/docusaurus/edit/master/website/',
                    disableVersioning: false,
                    includeCurrentVersion: true,
                    versions: {
                        current: {
                            label: 'Devex Main',
                        },
                        '3.10.2': {
                            label: 'Devex 3.10',
                            path: 'version-3.10.2',
                        },
                    },
                    onlyIncludeVersions: ['current', '3.10.2'],
                },
                blog: {
                    showReadingTime: true,
                    // Please change this to your repo.
                    editUrl:
                        'https://github.com/facebook/docusaurus/edit/master/website/blog/',
                },
                theme: {
                    customCss: require.resolve('./custom.css'),
                },
            },
        ],
    ],
```

how do i configure the navbar item to link the currently selected version of the documents?

**Comment by Josh-Cena** [comment]
The "current" version would take up the root route and would not have a version name in it. You can manually add one:

```js
current: {
    label: 'Devex Main',
    path: 'next'
},
```

**Comment by Sotty75** [comment]
hi, 
i modified it but i still have the same issue.
i have a feeling the problem is in the to:'docs' setting of the navbar item, the link is always pointing to that value, so i guess there is a way to make it "dynamic".
i would expect the link to dynamically point to the path of the currently selected version, like in the docusaurus site itself.
Any other suggestion?

```
items: [
                {
                    to: 'docs',
                    activeBasePath: 'System Requirements',
                    label: 'Administrative Guide',
                    position: 'left',
                },
```

**Reply by Josh-Cena** [reply]
Ah, so you mean that the routes are correct but the docs link is not pointing to the right version? Does it update correctly when you are browsing `docs/version-3.10.2/`, or is it still pointing to `docs/`?

Just pointing out: your `activeBasePath: 'System Requirements',` configuration doesn't make sense to me, it should be the base path on which you expect this item to be highlighted, e.g., on all subroutes of `docs/api/`. But that should be irrelevant

**Comment by Sotty75** [comment, marked as answer]
i resolved the problem replacing the item to point to an actual doc in the docs folder.

{
                    docId: '_index',
                    label: 'Administrative Guide',
                    position: 'left',
                    type: 'doc',
                },

**Reply by Josh-Cena** [reply]
Ugh, I totally missed it!

Yes, the `to: 'docs'` just creates a regular link that points to `docs/`, it doesn't do any magic with docs versions. You need the `type: 'doc'` to do so

**Comment by Sotty75** [comment]
thanks anyway! :)
